conf/env_files/designsafe.sample.env

```diff
@@ -188,3 +188,12 @@ MONGO_USER=
 MONGO_PASS=
 MONGO_HOST=
 MONGO_DB=
+
+##
+# Miscellaneous
+#
+
+DATACITE_USER=
+DATACITE_PASS=
+DATACITE_URL=
+DATACITE_SHOULDER=
```

Comment:
String not required. ([Why mention that?](https://tacc-team.slack.com/archives/C70D449U5/p1728514305873259?thread_ts=1728514192.725279&cid=C70D449U5))